.github/workflows/publish.yml

```diff
@@ -17,10 +17,15 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: default
-          toolchain: 1.66.1
+          toolchain: 1.73.0
           target: wasm32-wasi
           default: true
 
+      - name: IDK why i need to do this
```

Comment:
would remove this so it's not in the side as title as a step, as true as it may be

Reply:
forgot to delete